src/main/java/ledger/user_interface/ui_controllers/MainPageController.java

```diff
@@ -58,55 +51,39 @@ public void initialize(URL fxmlFileLocation, ResourceBundle resources) {
             try {
```

Comment:
This method is getting really long. Can we extract the lambda's so easier to read.